### DISCUSSION TITLE
Codici IVA presenti su Fatture in Cloud

### DISCUSSION THREAD

**Discussion by Alexxia5** [fattureincloud/api#423] [Q&A] [answered]
Ciao, 
avrei bisogno di sapere se il codice id presente nella lista delle voci IVA che sono visibili su Fatture In Cloud tramite la chiamata List Vat Types sono definitivi  e possono essere utilizzati per identificare un determinato valore.
Ad esempio se devo inserire per una voce in fattura l'iva al 22%, posso recuperarla filtrando per id='0'?
Se invece ho necessità di esporre l'iva allo 0% dei "Fuori campo IVA" posso utilizzare l'id='9'? 

Cosa mi consigliate di fare?
Grazie

Mi riferisco al campo id della lista che contiene 66 voci in totale.
{
  "data": [
   {
      "id": 0,
      "value": 22,
      "description": "",
      "notes": "",
      "e_invoice": true,
      "ei_type": "0",
      "ei_description": "",
      "is_disabled": false,
      "editable": false,
      "default": true
    },
    {
      "id": 1,
      "value": 21,
      "description": "",
      "notes": "",
      "e_invoice": true,
      "ei_type": "0",
      "ei_description": "",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 2,
      "value": 20,
      "description": "",
      "notes": "",
      "e_invoice": true,
      "ei_type": "0",
      "ei_description": "",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 3,
      "value": 10,
      "description": "",
      "notes": "",
      "e_invoice": true,
      "ei_type": "0",
      "ei_description": "",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 4,
      "value": 4,
      "description": "",
      "notes": "",
      "e_invoice": true,
      "ei_type": "0",
      "ei_description": "",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 6,
      "value": 0,
      "description": "",
      "notes": "",
      "e_invoice": true,
      "ei_type": "4",
      "ei_description": "",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 7,
      "value": 0,
      "description": "Regime dei minimi",
      "notes": "Prestazione svolta in regime fiscale di vantaggio ex art. 1, commi 96-117, legge 24/12/2007 n&deg; 244 del 2007 come modificata dall'art. 27, DL 98/2011 e pertanto non soggetta a IVA ai sensi del provvedimento n. 185820/2011 dell'Agenzia delle Entrate",
      "e_invoice": true,
      "ei_type": "2.2",
      "ei_description": "Regime fiscale di vantaggio ex art. 1, commi 96-117, legge 24/12/2007 n. 244",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 9,
      "value": 0,
      "description": "Fuori campo IVA",
      "notes": "",
      "e_invoice": true,
      "ei_type": "2.2",
      "ei_description": "Fuori campo IVA",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 10,
      "value": 0,
      "description": "Oper. non soggetta, art.7 ter",
      "notes": "Operazione non soggetta a IVA ai sensi dell'art.7 ter D.P.R. 633/72",
      "e_invoice": true,
      "ei_type": "2.1",
      "ei_description": "Non soggetta art.7 ter DPR 633/72",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 11,
      "value": 0,
      "description": "Inversione contabile, art.7 ter",
      "notes": "Inversione contabile ai sensi dell'art.7 ter D.P.R. 633/72",
      "e_invoice": true,
      "ei_type": "6.9",
      "ei_description": "Inversione contabile art.7 ter DPR 633/72",
      "is_disabled": false,
      "editable": false
    },
    {
      "id": 12,
      "value": 0,
      "description": "Non Imponibile",
      "notes": "Non imponibile ai sensi dell'art.15 D.P.R. 633/72",
      "e_invoice": true,
      "ei_type": "1",
      "ei_description": "Escl. art. 15 DPR 633/72",
      "is_disabled": false,
      "editable": false
    },
...

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Alexxia5 
i campi filtrabili sono elencati qui: https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results

Sì, quegli sono gli ID dei valori di IVA preimpostati, quindi può procedere tranquillamente al loro utilizzo all'interno delle richieste di creazione dei documenti.

Grazie
Mauro

**Reply by Alexxia5** [reply]
Grazie mille @valmoz. 

Ciao Alex
